gubernator/log_parser.py

```diff
@@ -64,7 +64,10 @@ def digest(data, skip_fmt=lambda l: '... skipping %d lines ...' % l,
             previous_end = 0
         skip_amount = match - previous_end - CONTEXT
         if skip_amount > 1:
-            output.append('<span class="skip">%s</span>' % skip_fmt(skip_amount))
+            output.append('<span class="skip"><a href="javascript:toggle_skipped(%s)">%s</a></span>' % (match, skip_fmt(skip_amount)))
+            output.append('<div id="%s" style = "display:none;"><p>' % match)
```

Comment:
Add a `class` to the skipped lines. We may want to show them in a different color.